tests/integration/pilot/common/routing.go

```diff
@@ -2454,7 +2456,7 @@ spec:
 				name:   "http cookie without ttl" + c.Config().Service,
 				config: svc + tmpl.MustEvaluate(cookieWithoutTTLDest, ""),
 				call:   c.CallOrFail,
-				opts:   cookieCallOpts,
+				opts:   cookieWithoutTTLCallOpts,
```

Comment:
what's the diff behavior do you want to test with no ttl?

Reply:
with no ttl - envoy does create cookie so we have to pass the cookie to get the consistent hash routing work

Reply:
Yeah, why both tests passed